hack/generate-manifest-flow-aggregator.sh

```diff
@@ -20,16 +20,17 @@ function echoerr {
     >&2 echo "$@"
 }
 
-_usage="Usage: $0 [--mode (dev|release)] [-fc|--flow-collector] [--keep] [--help|-h]
+_usage="Usage: $0 [--mode (dev|release)] [-fc|--flow-collector <addr>] [-ch|--clickhouse] [--keep] [--help|-h]
 Generate a YAML manifest for the Flow Aggregator, using Kustomize, and print it to stdout.
-        --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
-        --flow-collector      Flow collector is the externalFlowCollectorAddr configMap parameter
-                              It should be given in format IP:port:proto. Example: 192.168.1.100:4739:udp
-        --keep                Debug flag which will preserve the generated kustomization.yml
-        --coverage            Generate a manifest which supports measuring code coverage of the Flow Aggregator binaries.
-        --verbose-log         Generate a manifest with increased log-level (level 4) for the Flow Aggregator. 
-                              This option will work only with 'dev' mode.
-        --help, -h            Print this message and exit
+        --mode (dev|release)            Choose the configuration variant that you need (default is 'dev')
+        --flow-collector, -fc <addr>    Flow collector is the flowCollector address configMap parameter
```

Comment:
Change to: "Specify the flowCollector address."?

Add "." at the end of the sentence.

Reply:
Thanks Jianjun for the reviews. Addressed.